guides/v2.3/comp-mgr/cli/cli-upgrade.md

```diff
@@ -23,7 +23,7 @@ Instead, see [Update the Magento application] for upgrade instructions.
 Complete the following prerequisites to prepare your environment before starting the upgrade process:
 
 - **Complete the Update and upgrade checklist**—To avoid possible errors during installation or upgrading, complete the [Update and upgrade checklist].
-- **Set the `pub/` directory root**—If you set the Magento root directory to `pub/`, create another subdomain or docroot that uses the Magento installation directory as its root, and run the [System Upgrade utility] using that subdomain.
+- **Set the `pub/` directory root**—If you set the Magento root directory to `pub/`, create another subdomain or docroot that uses the Magento installation directory as its root and [modify it to enhance security]({{ page.baseurl }}/install-gde/tutorials/change-docroot-to-pub.html)..
```

Comment:
I'm not sure we need the following part at all:
"create another subdomain or docroot that uses the Magento installation directory as its root". 

In the current version, it's related to running System Upgrade Utility using UI. Since this part is not supposed to be related to the current topic (upgrade using CLI), it should be removed. 

So we should have something like:

**Set the `pub/` directory root**— Use [Modify docroot to improve security]({{ page.baseurl }}/install-gde/tutorials/change-docroot-to-pub.html) for the details.

Reply:
**fixed**

Thank you, @rogyar